docs/content/docs/tutorials/ec2-example.md

```diff
@@ -0,0 +1,450 @@
+---
+title: "Manage a VPC Workflow with the ACK EC2-Controller"
+description: "Create and manage a network topology using ACK EC2-Controller deployed on Amazon Elastic Kubernetes Service (EKS) The ACK service controller for Elastic Compute Cloud (EC2-Controller) lets users manage EC2 resources directly from Kubernetes. This guide demonstrates how to deploy a basic network topology (consisting of VPC resources) using a single Kubernetes resource manifest."
+lead: "Create and manage a basic network topology using ACK EC2-Controller."
+draft: false
+menu:
+  docs:
+    parent: "tutorials"
+weight: 43
+toc: true
+---
+ 
+
+## Setup
+Although it is not necessary to use Amazon Elastic Kubernetes Service (Amazon EKS) or Amazon Elastic Container Registry (Amazon ECR) with ACK, this guide assumes that you have 
+access to an Amazon EKS cluster. If this is your first time creating an Amazon EKS cluster and Amazon ECR repository, see [Amazon EKS Setup][eks-setup] and [Amazon ECR Setup](https://docs.aws.amazon.com/AmazonECR/latest/userguide/get-set-up-for-amazon-ecr.html). 
+ 
+
+### Prerequisites
+ 
+This guide assumes that you have:
+ 
+- Created an EKS cluster with Kubernetes version 1.16 or higher.
+- Have access to Amazon ECR
+- AWS IAM permissions to create roles and attach policies to roles.
+- Installed the following tools on the client machine used to access your Kubernetes cluster:
+  - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv1.html) - A command line tool for interacting with AWS services.
+  - [kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html) - A command line tool for working with Kubernetes clusters.
+  - [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html) - A command line tool for working with EKS clusters.
+  - [Helm 3.8+](https://helm.sh/docs/intro/install/) - A tool for installing and managing Kubernetes applications.
+  - [Docker](https://docs.docker.com/engine/install/) - A tool to build, share, and run containers.
+ 
+ 
+### Install ACK EC2-Controller
+ 
+Deploy the EC2-Controller using the Helm chart, [ec2-chart](https://gallery.ecr.aws/aws-controllers-k8s/ec2-chart). Note, this example creates resources in the `us-west-2` region, but you can use any other region supported in AWS.
+ 
+* Log into the Helm registry that stores the ACK charts:
+ 
+```bash
+aws ecr-public get-login-password --region us-west-2 | helm registry login --username AWS --password-stdin public.ecr.aws
```

Comment:
i got the following error. what does this mean?
`An error occurred (UnsupportedCommandException) when calling the GetAuthorizationToken operation: GetAuthorizationToken command is only supported in us-east-1.`